velox/functions/prestosql/tests/CastBaseTest.h

```diff
@@ -67,25 +69,28 @@ class CastBaseTest : public FunctionBaseTest {
     assertEqualVectors(expected, result);
   }
 
+  core::TypedExprPtr buildCastExprDictEncoding(
+      const TypePtr& fromType,
+      const TypePtr& toType,
+      bool tryCast) {
+    core::TypedExprPtr inputField =
+        std::make_shared<const core::FieldAccessTypedExpr>(fromType, "c0");
+    core::TypedExprPtr callExpr = std::make_shared<const core::CallTypedExpr>(
+        fromType,
+        std::vector<core::TypedExprPtr>{inputField},
+        "testing_dictionary");
```

Comment:
nit: maybe rename to : "reverse_input_by_applying_dictionary_wrap"

Reply:
It's a bit long. What about "apply_reverse_dictionary"?

Reply:
Actually, could we rename this function in a separate diff? This function is used not only in my tests here.